src/main/java/com/google/sps/JobModel.java

```diff
@@ -102,24 +108,32 @@ private void getMetrics(Dataflow dataflowService) throws IOException, IllegalArg
 
         metricTime = jobMetric.getMetricTime();
```

Comment:
I don't know if it's just GitHub - but does the indentation increase slightly from this line onward?

Reply:
Yes, you are right. Thank you!

Reply:
Resolved in https://github.com/googleinterns/step229-2020/pull/15/commits/d93c9abd9592079a818b7ad7396acdd078cc1aab